src/main/java/strategy/activities/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package strategy.activities;
+
+public interface OperationHandler {
+    Integer executionOfOperation(Integer currentAction, Integer quantity);
```

Comment:
```suggestion
    void executeTransaction(FruitTransaction transaction);
```
Імплементації цього методу мають змінювати стан Storage. Кожна транзакцыя - це окрема одиниця, виконання якої змінює запаси на складі.